ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1.  What are PropTypes used for?
+  PropTypes are used to validate the properties coming into a component. The properties would need to be the correct type to pass the test.
+1.  What is a lifecycle event in React?
+  A lifecycle event is any change that takes a place within the app. For example, after the app is initially rendered the componentDidMount method will run. If any state has changed in this method it will be passed to the app and changes will be rerendered to the screen.
+1.  What is a Higher Order Component?
+  A HOC is a component that can wrap a component and push new props to that component. It also encapsulates the component so as to keep the props private to the rest of the app.
+1.  What are three different ways to style components in React?
+  You can use a css file, prop-types or use something like reactstrap.
```

Comment:
Almost, css file, inline styles, or css-in-js styling (like reactstrap or styled components)